packages/bot-skeleton/src/scratch/blocks/Binary/After Purchase/trade_again.js

```diff
@@ -28,12 +70,37 @@ Blockly.Blocks.trade_again = {
         };
     },
     restricted_parents: ['after_purchase'],
+    // onchange(event) {
+    //     const trade_again_block = this.workspace.getTradeAgainBlock();
+    //     console.log('this', this);
+    //     console.log('trade again block:', trade_again_block);
+    //     if (event.type === Blockly.Events.BLOCK_CHANGE) {
+    //         const SL_ENABALED = this.getFieldValue('SL_ENABALED');
+    //         let cc = trade_again_block.getDescendants();
+    //         console.log('cccc: ', cc);
+    //     }
+    // },
 };
 
-Blockly.JavaScript.trade_again = () => {
+Blockly.JavaScript.trade_again = block => {
+    block.error_message = 'ksjdkasjhdkajshd';
+    const is_sl_enabled = block.getFieldValue('SL_ENABALED').toLowerCase();
+    const is_tp_enabled = block.getFieldValue('TP_ENABALED').toLowerCase();
```

Comment:
Do we need to add toLowerCase() here?

Reply:
`block.getFieldValue(‘SL_ENABLED’) === ‘TRUE’ ? ‘true’ : ‘false’;` 
I replaced this. Thanks Mehrzad